ui/src/styles/style.css

```diff
@@ -145,8 +148,7 @@ tr:hover td {
 
 .button--small {
   width: 50%;
-  background-color:#80cbc4;
```

Comment:
Maybe we could change the background color here so it matches the rest of the app ?

Reply:
Or we could match the color of the texts using #af1e31 . Maybe making the background white and using the #af1e31 for the text would look good too :)

Reply:
@hgwood @Bugsyaya any idea ?

If not I believe you should choose @brunosabot since you did the UI :smile: